.ado/pipelines/templates/stages-chaos.yaml

```diff
@@ -8,10 +8,8 @@ parameters:
   - name: customPrefix
     type: string
   - name: testDurationSeconds
-    type: number
     default: 0
-  - name: loadTestStartSeconds
-    type: number
```

Comment:
I had to remove types in order to use variable values when calling this template... Locust load test has the same thing.